modules/rosa-hcp-sts-accessing-a-break-glass-cred-cli.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-accessing-a-break-glass-cred-cli_{context}"]
+= Accessing a {hcp-title} cluster by using a break glass credential
+
+Use the new `kubeconfig` from the break glass credential to gain temporary admin access to a {hcp-title} cluster.
+
+.Prerequisites
+
+* You have access to a {hcp-title} cluster with external authentication enabled. For more information, see _Creating a {hcp-title} cluster that uses external authentication_.
+* You have installed the `oc` CLI.
+
+.Procedure
+
+. Optional: If you have not already done so, create a `kubeconfigs` directory:
++
+[source,terminal]
+----
+$ mkdir ~/kubeconfigs
+----
++
+. Export the newly generated `kubeconfig` file, replacing <cluster_name> with the name of your cluster:
++
+[source,terminal]
+----
+$ export KUBECONFIG=~/kubeconfigs/break-glass-<cluster_name>.kubeconfig
+----
++
+. Access the details for the cluster:
++
+[source,terminal]
+----
+$ rosa describe break-glass-credential <break_glass_credential_id> -c <cluster_name>  --kubeconfig > $KUBECONFIG
+----
++
+. List the nodes from the cluster:
++
+[source,terminal]
+----
+$ oc get nodes
+----
+.Example output
++
+[source,terminal]
+----
+NAME                        STATUS   ROLES   AGE   VERSION
+ip-10-0-0-27.ec2.internal   Ready    worker  8m    v1.28.7+f1b5f6c
+ip-10-0-0-67.ec2.internal   Ready    worker  9m    v1.28.7+f1b5f6c
+----
+. Verify you have the correct credentials:
++
+[source,terminal]
+----
+$ kubectl auth whoami
+----
++
+.Example output
++
+[source,terminal]
+----
+ATTRIBUTE    VALUE
+Username     system:customer-break-glass:test-user
+Groups       [system:masters system:authenticated]
+----
+. Apply the `ClusterRoleBinding` for the groups defined in the external OIDC provider. The `ClusterRoleBinding` maps the `rosa-hcp-admins` group that is created in Microsoft Entra ID to a group in the {hcp-title} cluster.
++
+[source,terminal]
+----
+oc apply -f - <<EOF
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: rosa-hcp-admins
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: f715c264-ab90-45d5-8a29-2e91a609a895
+EOF
+----
++
+.Example output
++
+[source,terminal]
+----
+clusterrolebinding.rbac.authorization.k8s.io/rosa-hcp-admins created
+----
++
+[NOTE]
+====
+Once the `ClusterRoleBinding` has been applied, the {hcp-title} cluster is configured, and the `rosa` CLI and the {hybrid-console-url} is authenticated through the external OpenID Connect (OIDC) provider. You can now start assigning roles and deploying applications on the cluster.
```

Comment:
Per ISG do not use _once_ to mean _after_.
```suggestion
After the `ClusterRoleBinding` has been applied, the {hcp-title} cluster is configured, and the `rosa` CLI and the {hybrid-console-url} is authenticated through the external OpenID Connect (OIDC) provider. You can now start assigning roles and deploying applications on the cluster.
```

Reply:
Is this change right?
```suggestion
Once the `ClusterRoleBinding` has been applied, the {hcp-title} cluster is configured, and the `rosa` CLI and the {hybrid-console-url} are authenticated through the external OpenID Connect (OIDC) provider. You can now start assigning roles and deploying applications on the cluster.
```

Reply:
Yes, thanks.